src/extension.ts

```diff
@@ -367,16 +369,21 @@ class ChatGPTViewProvider implements vscode.WebviewViewProvider {
 				</style>
 			</head>
 			<body>
-				<input class="h-10 w-full text-white bg-stone-700 p-4 text-sm" placeholder="Ask ChatGPT something" id="prompt-input" />
-				
-				<div id="response" class="pt-4 text-sm">
-				</div>
-
-				<script src="${scriptUri}"></script>
+				<div id="root"></div>
+				<script type="module" nonce="${nonce}" crossorigin src="${scriptUri}"></script>
 			</body>
 			</html>`;
 	}
 }
 
 // This method is called when your extension is deactivated
-export function deactivate() {}
\ No newline at end of file
+export function deactivate() {}
+
+function getNonce() {
+	let text = "";
+	const possible = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789";
+	for (let i = 0; i < 32; i++) {
+		text += possible.charAt(Math.floor(Math.random() * possible.length));
+	}
+	return text;
+}
```

Comment:
@jbbernardin basically this function looks at all the possible characters and randomly chooses 32 of them to create the unique identifier.